airflow/www/views.py

```diff
@@ -271,7 +271,12 @@ def get_date_time_num_runs_dag_runs_form_data(www_request, session, dag):
         base_date = (date_time + datetime.timedelta(seconds=1)).replace(microsecond=0)
 
     default_dag_run = conf.getint("webserver", "default_dag_run_display_number")
-    num_runs = www_request.args.get("num_runs", default=default_dag_run, type=int)
```

Comment:
The change was made to ensure compatibility across different environments and setups, particularly in testing scenarios where request.args might not support keyword arguments. For me, I obtained a Keyword error while trying to create my mock data. Although this does the same operation, some formdata does not support keyword arguments. Correct me if I'm wrong tho :D

Reply:
Ideally you don't want to change the code to fit the test, you'd be better off changing the test to fit the code. I saw in the test you were creating a custom class, it might be better to use the class that's actually passed into the method. Perhaps some of the other tests might be using it, that should help you out.

Reply:
I see, how can I find the class that was passed into the method? Thanks!

Reply:
I think I got it figured out. We can instead use response to pass it in as it's used via API call.

Reply:
Got it fixed in my other commit. Boosted code coverage to 85%